cmd/server/main.go

```diff
@@ -2,15 +2,32 @@ package main
 
 import (
 	"todoapp/internal/adapter/http/rest"
-	"todoapp/internal/adapter/memdb"
+	"todoapp/internal/adapter/mssqldb"
 	"todoapp/internal/common/config"
+	"todoapp/internal/entity"
+
+	"gorm.io/driver/sqlserver"
+	"gorm.io/gorm"
 )
 
 func main() {
-	userRepo := memdb.NewUserRepo()
-	todoRepo := memdb.NewTodoRepo()
+	// userRepo := memdb.NewUserRepo()
+	// todoRepo := memdb.NewTodoRepo()
+	dsn := "sqlserver://sa:MyStrongPassword123@localhost:1433?database=todoapp"
```

Comment:
"Never ever commit sensitive information" to git repo. If this is a db for local dev only, then make these info as default, and the real value should be injected in, as env variables.

Break this link into these parts:
- URL template
- Username
- Password
- Database host
- Database name

Suggested: config package.

Reply:
one more thing: database port too